modules/models/src/main/java/org/eclipse/xpanse/modules/models/credential/AbstractCredentialInfo.java

```diff
@@ -87,4 +87,8 @@ public abstract class AbstractCredentialInfo {
         this.description = description;
```

Comment:
can you also please update the description of "name" field with more information?